### ISSUE TITLE
`tiup playground nightly` does not work because prometheus version is not found

### ISSUE THREAD

**Issue by sticnarf** [pingcap/tiup#1531]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

```bash
$ tiup playground nightly
```

2. What did you expect to see?

A nightly TiDB cluster should start successfully.

3. What did you see instead?

```
$ tiup playground nightly
Starting component `playground`: /home/yilin/.tiup/components/playground/v1.5.5/tiup-playground nightly
Using the version v5.0.0-nightly-20210820 for version constraint "nightly".

If you'd like to use a TiDB version other than v5.0.0-nightly-20210820, cancel and retry with the following arguments:
    Specify version manually:   tiup playground <version>
    Specify version range:      tiup playground ^5
    The nightly version:        tiup playground nightly

Playground Bootstrapping...
The component `prometheus` version v5.0.0-nightly-20210820 is not installed; downloading from repository.
Error: Playground bootstrapping failed: version v5.0.0-nightly-20210820 on linux/amd64 for component prometheus not found: unknown version
Error: run `/home/yilin/.tiup/components/playground/v1.5.5/tiup-playground` (wd:/home/yilin/.tiup/data/Sgc3gx9) failed: exit status 1
```

4. What version of TiUP are you using (`tiup --version`)?

```
$ tiup --version
1.5.5 tiup
Go Version: go1.16.7
Git Ref: v1.5.5
GitHash: 01df6de49dbf23af1a3e9d6f71d383c407cdb810
```

**Comment by sticnarf** [comment]
Is it possible to find the latest available version which contains all necessary components (like rustup)?

**Comment by qqqdan** [comment]
It has risk to auto install an "unexpected" version. It is better to fix from CI side.

**Comment by qqqdan** [comment]
will fix in https://github.com/pingcap/tiup/issues/1713

**Comment by qqqdan** [comment]
will fix in 1.9.0
